tests/python/unittest/test_meta_schedule_tune_tir.py

```diff
@@ -91,127 +91,6 @@ def test_tune_matmul_cuda():
             print(sch.trace)
 
 
-@pytest.mark.skip("Integeration test")
```

Comment:
May I ask why this test is removed?

Reply:
This test depends on auto-tensorization for tensorcore, which is not in `main`

Reply:
I see. We may upstream it later then.

Reply:
Ooops...Thanks for spotting this!